v22.2/cloud-storage-authentication.md

```diff
@@ -0,0 +1,500 @@
+
+[The JSON credentials file for authentication](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console) can be downloaded from the **Service Accounts** page in the Google Cloud Console and then [base64-encoded](https://tools.ietf.org/html/rfc4648):
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+cat gcs_key.json | base64
+~~~
+
+Include `AUTH=specified` and pass the encoded JSON object to the `CREDENTIALS` parameter:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+BACKUP DATABASE <database> INTO 'gs://{bucket name}/{path}?AUTH=specified&CREDENTIALS={encoded key}';
+~~~
+
+## Google Cloud Storage implicit
+
+{{site.data.alerts.callout_info}}
+`implicit` authentication **cannot** be used to run bulk operations from {{ site.data.products.db }} clusters—instead, use `AUTH=specified`.
+{{site.data.alerts.end}}
+
+For CockroachDB instances that are running within a Google Cloud Environment, [environment data](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application) can be used from the [service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to implicitly access resources within the storage bucket.
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+For CockroachDB clusters running in other environments, `implicit` authentication access can still be set up manually with the following steps:
+
+  1. [Create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) and add the service account address to the permissions on the specific storage bucket.
+
+  1. Download the [JSON credentials file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console) from the **Service Accounts** page in the Google Cloud Console to the machines that CockroachDB is running on. (Since this file will be passed as an environment variable, it does **not** need to be base64-encoded.) Ensure that the file is located in a path that CockroachDB can access.
+
+  1. Create an environment variable instructing CockroachDB where the credentials file is located. The environment variable must be exported on each CockroachDB node:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    export GOOGLE_APPLICATION_CREDENTIALS="/{cockroach}/gcs_key.json"
+    ~~~
+
+    Alternatively, to pass the credentials using [`systemd`](https://www.freedesktop.org/wiki/Software/systemd/), use `systemctl edit cockroach.service` to add the environment variable `Environment="GOOGLE_APPLICATION_CREDENTIALS=gcs-key.json"` under `[Service]` in the `cockroach.service` unit file. Then, run `systemctl daemon-reload` to reload the `systemd` process. Restart the `cockroach` process on each of the cluster's nodes with `systemctl restart cockroach`, which will reload the configuration files.
```

Comment:
technical nit: `systemctl daemon-reload` does not reload the `systemd` process, but causes it to refresh the config of managed unit files, such as our `cockroach` one. The following avoids the discrepancy:

> Then, run `systemctl daemon-reload` to reload the `cockroach.service` unit file with the updated configuration.